package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nll/datum",
-  "version": "3.4.0",
+  "version": "3.5.0",
```

Comment:
Version numbers are managed with standard-version. If you don't mind, this could be reverted and on the subsequent release commit it will be bumped.